egs/librispeech/ASR/local/compute_fbank_librispeech.py

```
@@ -129,6 +129,10 @@ def compute_fbank_librispeech(
                 recordings=m["recordings"],
                 supervisions=m["supervisions"],
             )
+            
+            # Resample audio to 16kHz to match Fbank extractor's expected sampling rate
+            logging.info(f"Resampling audio to 16000 Hz")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)